src/Symfony/Component/Mailer/EventListener/MessageLoggerListener.php

```diff
@@ -14,7 +14,6 @@
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Mailer\Event\MessageEvent;
 use Symfony\Component\Mailer\Event\MessageEvents;
```

Comment:
Tu be removed

Reply:
Nope! (extra `s` :))

Reply:
Also, no threats please 😋
<img width="243" alt="Screenshot 2022-10-25 at 22 44 22" src="https://user-images.githubusercontent.com/7502063/197878172-96b87190-be90-41b7-913f-8314119a2311.png">

Reply:
Oh, my bad :p